.flowconfig

```diff
@@ -17,4 +17,6 @@
 [options]
 emoji=true
 exact_by_default=true
-module.use_strict=true
\ No newline at end of file
+module.use_strict=true
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe
+suppress_comment=\\(.\\|\n\\)*\\$FlowIgnore
```

Comment:
Added these suppress comments because I wanted the expressiveness of `$FlowIgnore`